File: python/test/test_constant.py
import ThrustRTC as trtc


darr = trtc.device_vector_from_list([3, 7, 2, 5 ], 'int32_t')
trtc.Transform_Binary(darr, trtc.DVConstant(trtc.DVInt32(10)), darr, trtc.Plus())
print (darr.to_host())

File: java/test/test_constant.java
import java.util.*;
import JThrustRTC.*;

public class test_constant
{
	public static void main(String[] args) 
	{
		DVConstant src = new DVConstant(new DVInt32(123), 10);
		DVVector dst = new DVVector("int32_t", 10);
		TRTC.Copy(src, dst);
		System.out.println(Arrays.toString((int[])dst.to_host()));
	}

}
